src/qutip_qtrl/pulsegen.py

```diff
@@ -1042,16 +1042,18 @@ def init_pulse(self, num_coeffs=None):
     #            self.guess_pulse = self.guess_pulsegen.gen_pulse()
     #        return self.guess_pulse
 
-    def init_coeffs(self, num_coeffs=None):
+    def init_coeffs(self, num_coeffs=None, init_coeffs=None):
         """
-        Generate the initial ceofficent values.
+        Generate or set the initial ceofficent values.
 
         Parameters
         ----------
         num_coeffs : integer
             Number of coefficients used for each basis function
             If given this overides the default and sets the attribute
             of the same name.
+        init_coeffs : float array[num_coeffs * num_basis_funcs]
+            Typically this will be the initial basis coefficients.
```

Comment:
```suggestion
            Typically this will be the initial basis coefficients.
            If set to `None` (the default), the initial coefficients will be automatically generated.
```